arches_modular_reports/src/arches_modular_reports/ModularReport/components/ChildTileNodeValue.vue

```diff
@@ -13,29 +13,28 @@ import type {
     URLDetails,
 } from "@/arches_modular_reports/ModularReport/types";
 
-const { nodeValue, userIsRdmAdmin = false } = defineProps<{
-    nodeValue: NodeData | null;
+const { value, userIsRdmAdmin = false } = defineProps<{
+    value: NodeData | null;
     userIsRdmAdmin?: boolean;
 }>();
 
 const { $gettext } = useGettext();
 
-const displayValue = computed(() => nodeValue?.display_value);
-const interchangeValue = computed(() => nodeValue?.interchange_value);
+const displayValue = computed(() => value?.display_value);
+const nodeValue = computed(() => value?.node_value);
+const details = computed(() => value?.details);
 </script>
 
 <template>
-    <dd v-if="nodeValue === null || interchangeValue === null">
+    <dd v-if="value === null || nodeValue === null">
         {{ $gettext("None") }}
     </dd>
     <div
-        v-else-if="
-            Array.isArray(interchangeValue) && interchangeValue[0]?.resource_id
-        "
+        v-else-if="(details as ResourceDetails[])[0]?.resource_id"
```

Comment:
`resource_id` looks like a required property of type `ResourceDetails`.  If so, can we remove this check.  I would assume that every tile (child or otherwise) would have a resource id.

Reply:
Ah, this is the template that renders links for a ResourceInstance{List}DataType node value, so only certain nodes will match this test.

Open to better names!